pkgs/games/gargoyle/default.nix

```
@@ -20,13 +20,13 @@ let
 in
 
 stdenv.mkDerivation {
-  name = "gargoyle-2018-10-06";
+  name = "gargoyle-2019-01-01";
```

```suggestion
  pname = "gargoyle";
  version = "2019.1.1";
```

I checked the revision and it actually seems to respond to the tag `2019.1.1`, so it seems to be better to use that as the release. Besides that the release date was Nov 18, 2019  ;).

This change is also a good excuse to switch to the newer `pname`/`version` convention.